### PR TITLE
Updated share portfolio modal to look more like mocks.

### PR DESCRIPTION
**Pull request by Hyperkid123**
### Changes
- update share portfolio look

https://projects.engineering.redhat.com/browse/SSP-350
https://marvelapp.com/625jh26/screen/59029719

#### Before
screenshot-ci cloud redhat com-2019 11 04-10_13_15

#### After
screenshot-ci foo redhat com_1337-2019 11 04-10_12_16

### CODE REVIEW

File: src/presentational-components/shared/loader-placeholders.js [hunk 1]
```diff
@@ -158,14 +160,19 @@ const FormItemLoader = () => (
...
     <FormGroup fieldId="1">
       <FormItemLoader />
     </FormGroup>
-    <FormGroup fieldId="2">
-      <FormItemLoader />
-    </FormGroup>
-    <Title size="xl">Groups with access</Title>
+    <TextContent>
+      <Text component={ TextVariants.small }>
+        Invite group
```
Comment (lgalis) [inline src/presentational-components/shared/loader-placeholders.js]: The text here should be 'Groups with access

**Comment by Hyperkid123** [comment]
@lgalis so i wanted to make the groups select async select. But the rbac api does not support all the filtering features we would need to make it work properly. I added the clearable option to the select so the value can be deleted instead of empty option.

**Comment by lgalis** [comment]
@Hyperkid123 - I do not see the clearable option:
Screenshot from 2019-11-05 09-20-04

**Comment by Hyperkid123** [comment]
@lgalis a forgot to push the change :man_facepalming:

**Pull request merged**